app/code/Magento/MediaGallery/Test/Unit/Model/DataExtractorTest.php

```diff
@@ -54,22 +53,41 @@ public function testExtractData(string $class, $interfaceClass, array $expectedD
                 'data' => $data,
             ]
         );
-        $receivedData = $this->dataExtractor->extract($model, $interfaceClass);
-        $this->checkValues($expectedData, $receivedData, $model);
+        if ($interfaceClass) {
+            $receivedData = $this->dataExtractor->extract($model, $interfaceClass);
+            $this->checkAssetValues($expectedData, $receivedData, $model);
+        } else {
+            $receivedData = $this->dataExtractor->extract($model);
+            $this->checkKeyWordValues($expectedData, $receivedData, $model);
+        }
+    }
+
+    /**
+     * @param array $expectedData
+     * @param array $data
+     * @param object $model
+     */
+    protected function checkAssetValues(array $expectedData, array $data, $model)
```

Comment:
I would suggest using `private` scope for the private methods. `protected` scope assumes that we consider an inheritance of this test class by another test class. It's not a big deal, however, it's recommended to be strict with scoping and hide implementation details in such cases.